docs/specification.yml

```diff
@@ -1785,17 +1785,20 @@ components:
       properties:
         isValid:
           type: boolean
-          description: true of false if valid. if false a detail key would be good to detail the error but not required
+          description: True or false if valid. If false, a detail key is included that details the error 
         detail:
-          type: object
-          description: details of the error if there was one.
+          description: details of the error if there were some
+          oneOf:
```

Comment:
Now that I look at the code, I think `detail` is always a `str`. There was some refactoring some time ago, and IFAIK, all the return messages only set a string value for detail.

Reply:
https://github.com/CSCfi/metadata-submitter/blob/55f2be308c4e74c522c1f075b7df4838f2b4f35c/metadata_backend/api/middlewares.py#L123

Reply:
Exception's reason is a `str` https://docs.aiohttp.org/en/stable/web_reference.html?highlight=exception#http-exceptions